src/lldp_syncd/daemon.py

```diff
@@ -367,18 +387,32 @@ def sync(self, parsed_update):
                 logger.debug("sync'd: {}".format(json.dumps(chassis_update, indent=3)))
 
         new, changed, deleted = self.cache_diff(self.interfaces_cache, parsed_update)
+
+        # For changed elements, if only lldp_rem_time_mark changed, update its value, otherwise delete and repopulate
+        for interface in changed:
+            if re.match(SONIC_ETHERNET_RE_PATTERN, interface) is None:
+                logger.warning("Ignoring interface '{}'".format(interface))
+                continue
+            table_key = ':'.join([LldpSyncDaemon.LLDP_ENTRY_TABLE, interface])
+            if self.is_only_time_mark_modified(self.interfaces_cache[interface], parsed_update[interface]):
+                self.db_connector.set(self.db_connector.APPL_DB, table_key, 'lldp_rem_time_mark', parsed_update[interface]['lldp_rem_time_mark'], blocking=True)
+                logger.debug("Only sync'd interface {} lldp_rem_time_mark: {}".format(interface, parsed_update[interface]['lldp_rem_time_mark']))
+            else:
+                self.db_connector.delete(self.db_connector.APPL_DB, table_key)
+                self.db_connector.hmset(self.db_connector.APPL_DB, table_key, parsed_update[interface])
+                logger.debug("Sync'd changed interface {} : {}".format(interface, parsed_update[interface]))
         self.interfaces_cache = parsed_update
-        # Delete LLDP_ENTRIES which were modified or are missing
-        for interface in changed + deleted:
+        # Delete LLDP_ENTRIES which are missing
+        for interface in deleted:
             table_key = ':'.join([LldpSyncDaemon.LLDP_ENTRY_TABLE, interface])
             self.db_connector.delete(self.db_connector.APPL_DB, table_key)
-        # Repopulate LLDP_ENTRY_TABLE by adding all changed elements
-        for interface in changed + new:
+            logger.debug("Delete table_key: {}".format(table_key))
+        # Repopulate LLDP_ENTRY_TABLE by adding new elements
+        for interface in new:
```

Comment:
The interface in new can merge with interface with changed? I mean line 392 can change to:

for interface in changed + new:
    .... set data with hmset here.

Reply:
@liuh-80 Per the offline discussion, there could be scenarios that changed interfaces have less element than before, hmset could just update the changed elements not remove non-existing ones. For example, the interface has 8 elements before, next time, it has only 7, hmset probably still keep 8 elements in database. 
Considering  this scenario, we agree to keep the current logic to make sure data is update accurately.